src/models/disassemblyoutput.cpp

```diff
@@ -120,6 +120,11 @@ QString findBinaryForSymbol(const QStringList& debugPaths, const QStringList& ex
     return {};
 }
 
+bool isHexCharacter(const QChar c)
+{
+    return (c >= QLatin1Char('0') && c <= QLatin1Char('9')) || (c >= QLatin1Char('a') && c <= QLatin1Char('f'));
```

Comment:
That's nice, especially the calling!
I just wasn't sure if there will be a 1 to 4 QLatin1Char objects created when entering this function which may be prevented when declaring them as static const in the function.

Reply:
no need to, the compiler can optimize it away [see](https://godbolt.org/z/9qxd1c1eM)